docs/JMS-batchsink.md

```diff
@@ -0,0 +1,57 @@
+# JMS Streaming Source
+
+
+Description
+-----------
+Produces JMS messages of different types as Message, Text, Bytes and Map to a specified Queue or Topic.
+
+Use Case
+--------
+Use this JMS Sink plugin when you want to produce messages to a JMS Queue/Topic. 
+
+
+Properties
+----------
+**Connection Factory**: Name of the connection factory. If not specified *ConnectionFactory* if considered by default.
+
+**JMS Username**: Username to connect to JMS. This property is mandatory.
+
+**JMS Password**: Password to connect to JMS. This property is mandatory.
+
+**Provider URL**: Provider URL of the JMS Provider. This property is mandatory. For example for the *ActiveMQ* provider
+ you can set: `tcp://hostname:61616`.
+
+**Type**: Queue or Topic. Queue is considered by default.
+
+**Destination**: Queue/Topic name.
```

Comment:
Is the topic created if its not preset?

Reply:
No, topic is not created if not present. This isn't specified in the PRD. Should I implement this @CuriousVini?